docker/puppetdb/Dockerfile

```diff
@@ -2,8 +2,9 @@ FROM ubuntu:16.04
 
 ARG vcs_ref
 ARG build_date
-ENV PUPPETDB_VERSION="5.2.2"
-ENV PUPPET_AGENT_VERSION="5.5.1"
+ARG version="5.1.5"
```

Comment:
Why an older version now?

Reply:
Because the team merged the initial work into the 5.1.x, so this is setting it to the latest version in the 5.1.x series